### DISCUSSION TITLE
How to debug goharbor

### DISCUSSION THREAD

**Discussion by consoleKEC** [goharbor/harbor#15862] [Q&A]
is there any document tell developers how to debug goharbor without container？

**Comment by jeyrce** [comment]
There is a container named `harbor-log` may help you, otherwise you can deploy a prometheus monitor system.

**Comment by consoleKEC** [comment]
> There is a container named `harbor-log` may help you, otherwise you can deploy a prometheus monitor system.

thanks, while what I want is not just log, I prefer a debug tools for develop in IDE tools or like GDB in linux environment.
